app/code/Magento/Ui/view/base/web/js/lib/core/collection.js

```diff
@@ -224,7 +224,7 @@ define([
         regionHasElements: function (name) {
             var region = this.getRegion(name);
 
-            return region().length > 0;
+            return region.length > 0;
         },
```

Comment:
any reason why we need to remove () ?

Reply:
ahh i see region is ObservableArray so we only need use length for counting 
Nice catch!

Reply:
I think that issue happens because there's an empty div in the widget as the following:

![Screenshot from 2020-04-29 13-02-26](https://user-images.githubusercontent.com/23085790/80589148-f0fab880-8a19-11ea-9c36-284f03b13de7.png)

So, I tracked the reason of that empty div and i found that:
1. the regionHasElements method always passes the condition
2.` console.log(region().length)` printing 1
3. after removing the parentheses, it returns the real length which is 0 in that case and the empty div has gone.